jaeger/charts/linkerd-jaeger/Chart.yaml

```diff
@@ -11,7 +11,7 @@ kubeVersion: ">=1.21.0-0"
 name: linkerd-jaeger
 sources:
 - https://github.com/linkerd/linkerd2/
-version: 30.11.0
+version: 30.11.0-edge
```

Comment:
You do need to bump the patch at least, because according to semver any version with a suffix comes before the same one without.